srtcore/core.cpp

```diff
@@ -5487,11 +5484,21 @@ SRT_REJECT_REASON CUDT::setupCC()
         // At this point we state everything is checked and the appropriate
         // corrector type is already selected, so now create it.
         HLOGC(pflog.Debug, log << "filter: Configuring: " << m_config.m_PacketFilterConfig.c_str());
-        if (!m_PacketFilter.configure(this, &(m_pRcvQueue->m_UnitQueue), m_config.m_PacketFilterConfig.str()))
+        bool status = true;
+        try
         {
-            return SRT_REJ_FILTER;
+            // The filter configurer is build the way that allows to quit immediately
+            // exit by exception, but the exception is meant for the filter only.
+            status = m_PacketFilter.configure(this, &(m_pRcvQueue->m_UnitQueue), m_config.m_PacketFilterConfig.str());
+        }
+        catch (CUDTException& )
```

Comment:
```suggestion
        catch (const CUDTException& )
```

Reply:
What does `const` change here if you don't have a designated variable?